src/ui/components/map/providers/mapboxmapprovider.js

```diff
@@ -11,6 +16,14 @@ import DOM from '../../../dom/dom';
  * @extends MapProvider
  */
 export default class MapBoxMapProvider extends MapProvider {
+  constructor (opts = {}, systemOpts = {}) {
+    super(opts, systemOpts);
+
+    this._language = SUPPORTED_LANGUAGES.includes(this._language)
+      ? this._language
+      : this._languageFallback;
```

Comment:
Are we going to need more sophisticated `languageFallback` logic than just choosing `en`? I think we may want to discuss further with Product.

Reply:
It might not be necessary. This was added for languages such as Italian which aren't supported by Mapbox. If Product wants to use a map language other than english for an Italian site then can specify a locale in a jambo VTC.

Reply:
Update: Product said to just fallback on english in this case